package.json

```diff
@@ -47,7 +47,7 @@
         "@angular/platform-browser": "^17.0.7",
         "@angular/router": "^17.0.7",
         "@egjs/hammerjs": "^2.0.17",
-        "@geoengine/openapi-client": "0.0.2",
+        "@geoengine/openapi-client": "https://gitpkg.now.sh/geo-engine/openapi-client/typescript?search-handler",
```

Comment:
This needs to be changed before merging

Reply:
[3a7dd57](https://github.com/geo-engine/geoengine-ui/pull/691/commits/3a7dd5705804d07e392452797fc053247cabd169)

Reply:
[d9df4f6](https://github.com/geo-engine/geoengine-ui/pull/691/commits/d9df4f679f3840be14bb7395bf524aeddaa93477)